docs/config_entries_config_flow_handler.md

```diff
@@ -522,6 +522,71 @@ class ExampleFlow(ConfigFlow):
         )
 ```
 
+## Use of SchemaConfigFlowHandler for simple flows
+
+For helpers and integrations with simple flows, you can use the `SchemaConfigFlowHandler` instead.
+
+All user input is stored in the `options` dictionary in your config entry, and therefore they could be made available in an options flow.
```

Comment:
I am not sure I understand this sentence since the `options` dict is also needed for the normal config flow, not only for the options flow.

Reply:
I have updated the text so hopefully we can work from there. Up for suggestions.